vaadin-date-time-picker-flow/src/test/java/com/vaadin/flow/component/datetimepicker/DateTimePickerSerializableTest.java

```diff
@@ -0,0 +1,30 @@
+package com.vaadin.flow.component.datetimepicker;
+
+import com.vaadin.flow.component.UI;
+import com.vaadin.flow.testutil.ClassesSerializableTest;
+
+import java.util.stream.Stream;
+
+public class DateTimePickerSerializableTest extends ClassesSerializableTest {
```

Comment:
Why does this class have a lot of stuff in it? For example `ComboBox` seems to have just an empty class.

Reply:
I don't know really. This was basically copied from DatePicker. Should this file just be removed or is it needed for something? Do we need some kind of "SerializableTest" (if so what should it do)?

Reply:
It seems that this UI needs to be initialized here, otherwise the serializable test also fails pointing to an NPE at `DatePicker` constructor here: https://github.com/vaadin/vaadin-date-picker-flow/blob/master/vaadin-date-picker-flow/src/main/java/com/vaadin/flow/component/datepicker/DatePicker.java#L88

where it calls `UI.getCurrent().getLocale()`.